scripts/build_utils.sh

```diff
@@ -946,6 +946,12 @@ build_debs() {
             egrep "*\.(changes|deb|ddeb|dsc|gz)$" | \
             egrep -v "(Packages|Sources|Contents)" | \
             $venv/chacractl binary ${chacra_flags} create ${chacra_endpoint}
+
+	# extract cephadm binary and push
+	dpkg-deb --fsys-tarfile release/${vers}/cephadm_${vers}*.deb | tar -x -f - --strip-components=3 ./usr/sbin/cephadm
+	echo cephadm | $venv/chacractl binary ${chacra_flags} create ${chacra_endpoint}
+	rm ./cephadm
```

Comment:
`rm ./cephadm` on deb, but not rpm?

Reply:
indecision, but inconsistency is the worst result.  I'll remove the rm, as leaving a crumb trail might be useful one day (until the build is purged)